confidential-data-hub/hub/protos/api.proto

```diff
@@ -18,10 +18,22 @@ message GetResourceResponse {
     bytes Resource = 1;
 }
 
+message KeyProviderKeyWrapProtocolInput {
+    bytes KeyProviderKeyWrapProtocolInput = 1;
+}
+
+message KeyProviderKeyWrapProtocolOutput {
+    bytes KeyProviderKeyWrapProtocolOutput = 1;
+}
+
 service SealedSecretService {
     rpc UnsealSecret(UnsealSecretInput) returns (UnsealSecretOutput) {};
 }
 
 service GetResourceService {
     rpc GetResource(GetResourceRequest) returns (GetResourceResponse) {};
 }
+
+service KeyProviderService {
+    rpc UnWrapKey(KeyProviderKeyWrapProtocolInput) returns (KeyProviderKeyWrapProtocolOutput) {};
```

Comment:
unwrap is one word, so `W` should probably be lowercase, but I don't think it's worth changing

Reply:
Well, the api name is defined by ocicrypt here https://github.com/containers/ocicrypt/blob/ffc163c18e970386fcd1b35acf2d350c8665d080/utils/keyprovider/keyprovider.proto#L16C1-L16C1

Reply:
😞 oh well